README.md

```diff
@@ -173,6 +175,70 @@ class { 'sensu':
 }
 ```
 
+### Installing Plugins
```

Comment:
If you can specify plugins and extensions as absent, this should be `### Managing Plugins`

Reply:
Added docs about setting `absent` for both plugins and extensions.